hud/init.lua

```diff
@@ -32,8 +32,8 @@ local function save()
     powerDisplayUsers = {}
     toolbarUsers = {}
     for address, data in pairs(glassData) do
-        if data.energyDisplay then table.insert(powerDisplayUsers, {component.proxy(address), {data.xRes, data.yRes}, data.scale, data.borderColor, data.primaryColor, data.accentColor}) end
-        if data.toolbar then table.insert(toolbarUsers, {component.proxy(address), {data.xRes, data.yRes}, data.scale, data.offset, data.borderColor, data.primaryColor, data.accentColor}) end
+        if data.energyDisplay then table.insert(powerDisplayUsers, {component.proxy(address), {data.xRes or 2560, data.yRes or 1440}, data.scale or 3, data.backgroundColor or colors.darkGray, data.primaryColor or colors.electricBlue, data.accentColor or colors.accentColor}) end
+        if data.toolbar then table.insert(toolbarUsers, {component.proxy(address), {data.xRes, data.yRes}, data.scale, data.offset, data.backgroundColor, data.primaryColor, data.accentColor}) end
```

Comment:
Didn't they have a default value somewhere else?

Reply:
They do, this is spaghetti code that fixed a crash I was having that I never properly fixed.